src/server/tran_server.hpp

```diff
@@ -178,7 +178,8 @@ class tran_server
       private:
 	void set_connection (cubcomm::channel &&chn);
 	// The default error handler for sending reqeust
```

Comment:
```suggestion
	// The default error handler for send/recv reqeust
```

Reply:
Actually, it's only for send (push_request). In the case of send_recv, the sending error handler is pushed per-request-based. See request_sync_client_server::send_recv(). An error handler is pushed with request.